ui/src/app/pages/apps-routes/app-available-show/app-available-show.page.html

```diff
@@ -54,6 +54,9 @@ <h3>
     </ion-button>
 
     <div *ngIf="vars.versionInstalled && vars.status !== 'INSTALLING' ">
+      <ion-button *ngIf="vars.versionInstalled" class="main-action-button" expand="block" fill="outline" [routerLink]="['/services', 'installed', vars.id]">
```

Comment:
This conditional is redundant. `vars.versionInstalled` already satisfied in outer scope.